Tasks/DeployVisualStudioTestAgent/README.md

```diff
@@ -9,6 +9,19 @@ The task requires:
 - .NET 4.5 on Windows7 Sp2 or Windows 2K8R2
 - Test machines should have PSRemoting enabled (run 'Enable-PSRemoting' on Windows Powershell)
 
+### Windows Remote Management (WinRM) Setup for On-premises Physical or Virtual Machines
+
+This task uses the [Windows Remote Management](https://msdn.microsoft.com/en-us/library/aa384426.aspx) (WinRM) to access domain-joined or workgroup, on-premises physical or virtual machines. 
+
+To easily **setup WinRM** on the **host machines** follow the directions for [domain-joined machines](https://www.visualstudio.com/en-us/docs/release/examples/other-servers/net-to-vm) or the [workgroup machines](https://www.visualstudio.com/en-us/docs/release/examples/other-servers/net-to-workgroup-vm).
+
+### Windows Remote Management (WinRM) Setup for Azure Virtual Machines
```

Comment:
@azooinmyluggage Do we really want to add these as different headings as it might confuse the readers.

Can't we have something like following:


 ###Prerequisites
 The task requires:
  - .NET 4.5 on Windows7 Sp2 or Windows 2K8R2
  - Test machines should have PSRemoting enabled (run 'Enable-PSRemoting' on Windows Powershell). In order to easily **setup WinRM** on the **test machines**, refer to following based on type of machine you want to use as test machines:
            1. Domain Joined On-premises Physical or Virtual Machines - Follow the directions on https://www.visualstudio.com/en-us/docs/release/examples/other-servers/net-to-vm
            2. Workgroup Machines On-premises Physical or Virtual Machines - Follow the directions on https://www.visualstudio.com/en-us/docs/release/examples/other-servers/net-to-workgroup-vm
            3. Azure Virtual Machines  - Azure virtual machines only work with the WinRM HTTPS protocol.
            i) Classic Azure VM - description , (https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-windows-tutorial-classic-portal/)
            ii) Existing Classic Azure VM - description
            iii) Resource Group - desription